.travis.yml

```diff
@@ -2,13 +2,11 @@ language: node_js
 
 node_js:
   - "6"
-#  - "8"
 
-env:
-  - RUN_PHANTOMJS=0
-  - RUN_PHANTOMJS=1
+before_script:
```

Comment:
Why remove PhantomJS testing and make these changes? These test cases are important to people running on older versions of JavaScript and in limited environments (like React Native).